web/src/validators/isThirdPerson.js

```diff
@@ -0,0 +1,8 @@
+// custom vuelidate validator to ensure input does not contain first person
+const isThirdPerson = (value) => {
```

Comment:
The naming is confusing from a grammar perspective (which includes 1st, 2nd, and 3rd, singular and plural in English), I would just call this isFirstPerson and negate it later on. The file will also need renaming, and the tests 😑 ... sorry it's a lot of flipping.
```suggestion
const isFirstPerson = (value) => {
```